src/openfermion/hamiltonians/_jellium_test.py

```diff
@@ -418,3 +418,20 @@ def test_plane_wave_energy_cutoff(self):
 
         max_diff = numpy.amax(numpy.absolute(spectrum_1 - spectrum_2))
         self.assertGreater(max_diff, 0.)
+
+    def test_plane_wave_period_cutoff(self):
+        grid = Grid(dimensions=2, length=2, scale=1.0)
+        spinless = True
+        period_cutoff = 0.
+
+        hamiltonian_1 = FermionOperator()
+        jw_1 = jordan_wigner(hamiltonian_1)
+        spectrum_1 = eigenspectrum(jw_1)
+
+        hamiltonian_2 = jellium_model(grid, spinless, True, False, None, True,
```

Comment:
I'm a little confused by this test, the difference should definitely be larger than 0 for any deviation.  I guess that this will be changed later in integration testing? It might make more sense to test against only the potential operator and see if its 0, to remove contributions by the kinetic operator.

Reply:
This is really a placeholder for future real integration test, when we figure out the correct formula for dual basis cutoff, periodic_cutoff will be set to non-zero, and hamiltonian_1 will be replaced with a jellium model.
Added a TODO.